prime-router/src/main/kotlin/azure/RequestFunction.kt

```diff
@@ -181,7 +181,7 @@ abstract class RequestFunction(
      */
     @Throws(InvalidParameterException::class)
     internal fun getDummySender(schemaName: String?, formatName: String?): TopicSender {
-        val errMsgPrefix = "No client found in header so expected valid " +
```

Comment:
Small clarifications of the error messages now that we have different use cases to account for 🤷